src/components/IndividualArticle.jsx

```diff
@@ -10,7 +10,24 @@ const IndividualArticle = () => {
     const [article, setArticle] = useState({})
     const [isLoading, setIsLoading] = useState(true)
     const [comments, setComments ] = useState([])
-    
+    const [newComment, setNewComment ] = useState("");
+
+
+
+    const onSubmit = event => {
+        setIsLoading(true)
+        event.preventDefault()
+        setComments((prevComments) => {
+            const newComments = [{comment_id:Date.now(), body: newComment, author: "grumpy19"},...prevComments]
+            return newComments;
+        })
+        postComment(article_id, "grumpy19", newComment).then((res) => {
+                setIsLoading(false)
+                return res
+        }).catch((err) => {
+                console.log(err)
```

Comment:
What should happen when something goes wrong and the comment isn't posted? The user is unlikely to check their browser console for errors! Right now if something went wrong it would look like it worked (because of the optimistic rendering) and they wouldn't get any indication it hadn't... Not ideal. What would you expect to happen as a user if something went wrong?